src/Orchard/Data/Migration/AutomaticDataMigrations.cs

```diff
@@ -30,6 +39,7 @@ public class AutomaticDataMigrations : IOrchardShellEvents {
 
         public void Activated() {
             IDistributedLock @lock;
+            EnsureDistributedLockSchema();

```

Comment:
Maybe name this one `EnsureDistributedLockSchemaExists()` and move it up one line?